arm-trafficmanager/2017-05-01/swagger/trafficmanager.json

```diff
@@ -0,0 +1,972 @@
+          "description": "The result of the operation or request."
+        }
+      },
+      "description": "The result of the request or operation."
+    },
+    "EndpointProperties": {
+      "properties": {
+        "targetResourceId": {
+          "type": "string",
+          "description": "Gets or sets the Azure Resource URI of the of the endpoint.  Not applicable to endpoints of type 'ExternalEndpoints'."
+        },
+        "target": {
+          "type": "string",
+          "description": "Gets or sets the fully-qualified DNS name of the endpoint.  Traffic Manager returns this value in DNS responses to direct traffic to this endpoint."
+        },
+        "endpointStatus": {
+          "type": "string",
+          "description": "Gets or sets the status of the endpoint..  If the endpoint is Enabled, it is probed for endpoint health and is included in the traffic routing method.  Possible values are 'Enabled' and 'Disabled'."
+        },
+        "weight": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the weight of this endpoint when using the 'Weighted' traffic routing method. Possible values are from 1 to 1000."
+        },
+        "priority": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the priority of this endpoint when using the ‘Priority’ traffic routing method. Possible values are from 1 to 1000, lower values represent higher priority. This is an optional parameter.  If specified, it must be specified on all endpoints, and no two endpoints can share the same priority value."
+        },
+        "endpointLocation": {
+          "type": "string",
+          "description": "Specifies the location of the external or nested endpoints when using the ‘Performance’ traffic routing method."
+        },
+        "endpointMonitorStatus": {
+          "type": "string",
+          "description": "Gets or sets the monitoring status of the endpoint."
+        },
+        "minChildEndpoints": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the minimum number of endpoints that must be available in the child profile in order for the parent profile to be considered available. Only applicable to endpoint of type 'NestedEndpoints'."
+        },
+        "geoMapping": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Gets or sets the list of countries/regions mapped to this endpoint when using the ‘Geographic’ traffic routing method. Please consult Traffic Manager Geographic documentation for a full list of accepted values."
+        }
+      },
+      "description": "Class representing a Traffic Manager endpoint properties."
+    },
+    "Endpoint": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the ID of the Traffic Manager endpoint."
```

Comment:
There should not be any `Gets` / `Sets` in the description. It's highly discouraged because that is very C# specific and when we generate documentation from here, customer doesn't like them. Let's make this better reference docs.